File: CPP/Algorithm/Selection_Sort.cpp
// Complexity: O(n^2)

int64_t aa[N], n;
void ssrt() {
    for (int i = 0; i < n - 1; ++i) {
        int mnin = i;
        for (int j = i + 1; j < n; ++j) {
            if (aa[j] < aa[mnin]) mnin = j;
        }
        if (i != mnin) swap(aa[i], aa[mnin]);
    }
}

File: Python/Algorithm/Selection_Sort.py
# Complexity: O(n^2)

def ssrt(ls):
    n = len(ls)
    for i in range(n - 1):
        mnin = i
        for j in range(i + 1, n):
            if ls[j] < ls[mnin]: mnin = j
        if i != mnin: ls[i], ls[mnin] = ls[mnin], ls[i]